Bottom: 91947b30d1e139a14f8730ebad2e6e9f66ba5002
Top:    f73fa4a329ab0487b3b74ed2ec0aa93aaffc0a0b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 02:01:23 +0100

Refresh of tmp

---

diff --git a/src/data.c b/src/data.c
index 7a8f575..3cf6231 100644
--- a/src/data.c
+++ b/src/data.c
@@ -57,8 +57,6 @@ BufferType Buffers[MAX_BUFFER];	/* my buffers */
 LibraryType Library;		/* the library */
 bool Bumped;			/* if the undo serial number has changed */
 
-LocationType Xorig, Yorig;	/* origin offset for drawing in pixmap */
-
 int addedLines;
 
 double Zoom_Multiplier = 0.01;
diff --git a/src/data.h b/src/data.h
index 6e70fd3..22eea0c 100644
--- a/src/data.h
+++ b/src/data.h
@@ -59,8 +59,6 @@ extern LibraryType Library;
 
 /*extern	DeviceInfoType	PrintingDevice[];*/
 
-extern LocationType Xorig, Yorig;
-
 extern double Zoom_Multiplier;
 
 extern char *Progname;
diff --git a/src/file.c b/src/file.c
index 6363bf4..5153a25 100644
--- a/src/file.c
+++ b/src/file.c
@@ -376,9 +376,6 @@ LoadPCB (char *Filename)
       Crosshair.Y =
 	MAX (0, MIN (PCB->CursorY, (LocationType) PCB->MaxHeight));
 
-      Xorig = Crosshair.X - TO_PCB (Output.Width / 2);
-      Yorig = Crosshair.Y - TO_PCB (Output.Height / 2);
-
       /* update cursor confinement and output area (scrollbars) */
       ChangePCBSize (PCB->MaxWidth, PCB->MaxHeight);
 
diff --git a/src/macro.h b/src/macro.h
index 6dcd0d1..625d639 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -53,9 +53,6 @@
 #ifndef	TO_PCB
 #define	TO_PCB(x)		((LocationType)((x)/Zoom_Multiplier))
 #endif
-#define	TO_PCB_X(x)		TO_PCB(x) + Xorig
-#define	TO_PCB_Y(y)		(SWAP_IDENT ? \
-				PCB->MaxHeight - TO_PCB(y) - Yorig : TO_PCB(y) + Yorig)
 
 /* ---------------------------------------------------------------------------
  * misc macros, some might already be defined by <limits.h>
